src/types/additionalProps.ts

```diff
@@ -29,4 +29,5 @@ export type GetNodeRefFunc = (node?: NodeAddress) => string | null;
 
 export interface AdditionalNodesProps extends Record<string, unknown> {
     getNodeRef?: GetNodeRefFunc;
+    database?: string;
 }
```

Comment:
Let pass `database` directly to components which use it and leave such interfaces for the props that are needed for multi-cluster version